src/lib/y2storage/callbacks/probe.rb

```diff
@@ -21,26 +19,14 @@
 # To contact SUSE LLC about this file by physical or electronic mail, you may
 # find current contact information at www.suse.com.
 
-require "storage"
+require "yast"
```

Comment:
why yast is required here? I do not see any Yast usage.

Reply:
Same than above. Yast usage is there, even if you don't see it. :wink: